### DISCUSSION TITLE
Tensor notation vs Voigt-Mandel notation

### DISCUSSION THREAD

**Discussion by andreichalapco** [Ferrite-FEM/Ferrite.jl#778] [General]
Hi,

I might be wrong but examining all the examples I saw that for the elasticity examples we use the Tensor notation to compute the elasticity tensor ... Is there also a possibility to do this in Mandel notation ... or is the design of the library made to work with just the tensor notation (using then Tensors.jl)? I'm asking this because I've might have missed something in the documentation and just wanted to be sure of that.

**Comment by koehlerson** [comment]
In the assembly you will likely invoke the shape gradient or symmetric shape gradient - depending on the deformation measure - which returns a `Tensors.Tensor` of second order. You can, of course, convert the return to a `Vector` that uses Mandel notation (and afterwards multiply it by your Mandel notation elasticity array). However, there is no advantage of doing so. Even worse, you are likely discarding performance optimizations that are done within the `Tensors` dispatched functions. So, yes, the library is more inclined to work with `Tensors` objects, however, there is no clear constraint and you can replace more or less every piece of it by whatever you like.

**Comment by andreichalapco** [comment]
Thx for quick reply. I was inclined to think the same thing, but I wanted to be sure! Very nice library btw :))
